src/main/java/homework_1/Main.java

```diff
@@ -1,9 +1,42 @@
 package homework_1;
 
-public class Main {
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
 
+public class Main {
   public static void main(String[] args) {
-    System.out.println("Hello homework!");
+    System.out.println("Current codepage: " + Charset.defaultCharset());
+    if (Charset.defaultCharset() != StandardCharsets.UTF_8) {
+      System.out.println("For correct display cyrillic args need to run with \"-Dfile.encoding=\"UTF-8\"\"");
+    }
+    try {
+      if (args.length == 0) {
+        System.out.println("No arguments provided!");
+      } else printOutput(args);
+    } catch (IOException e) {
+      System.err.println("Error!");
+      e.printStackTrace();
+    }
   }
 
-}
+  private static void printOutput(String[] inputArgs) throws IOException {
```

Comment:
To be honest, your code looks overcomplicated for the simple task. I would suggest simplifying it

Reply:
ok, i did it